lib/linalg/BasisGenerator.h

```diff
@@ -263,6 +283,20 @@ class BasisGenerator
         return d_svd->getNumSamples();
     }
 
+    /**
+     * @brief Prints the summary of recommended numbers of basis vectors.
+     *
+     * @param[in] energyFraction   Energy Fraction.
+     * @param[in] cutoff           Number of basis vectors selected.
+     * @param[in] cutoffOutputPath Path of the summary file.
+     * @param[in] first_sv         First singular vector in the calculaton of energy.
+     */
+    void finalSummary(
+        const double energyFraction,
```

Comment:
Should we input `1 - energyFraction` instead, with an appropriate name? It would be easier to put 1e-6 than 0.999999.